lib/puppet/graph/simple_graph.rb

```diff
@@ -225,18 +226,19 @@ def report_cycles_in_graph
     return if n == 0
     s = n == 1 ? '' : 's'
 
-    message = "Found #{n} dependency cycle#{s}:\n"
+    message = n_("Found %{num} dependency cycle:\n", "Found %{num} dependency cycles:\n", n) % { num: n }
     cycles.each do |cycle|
       paths = paths_in_cycle(cycle)
       message += paths.map{ |path| '(' + path.join(" => ") + ')'}.join("\n") + "\n"
     end
 
     if Puppet[:graph] then
       filename = write_cycles_to_graph(cycles)
-      message += "Cycle graph written to #{filename}."
+      message += _("Cycle graph written to %{filename}.") % { filename: filename }
     else
-      message += "Try the '--graph' option and opening the "
-      message += "resulting '.dot' file in OmniGraffle or GraphViz"
+      message += _("Try the '--graph' option and opening the ")
```

Comment:
--graph is a keyword, and we should leave a comment to leave it out of translation